phoenix-core/src/it/java/org/apache/phoenix/end2end/LogicalTableNameBaseIT.java

```diff
@@ -530,13 +531,16 @@ protected void validateIndex(Connection connection, String tableName, boolean is
     }
 
     public static void renameAndDropPhysicalTable(Connection conn, String tenantId, String schema, String tableName, String physicalName, boolean isNamespaceEnabled) throws Exception {
+        // if client is validating last_ddl_timestamp, this change in physical table name should be visible to the client
```

Comment:
Do we need this change given that default UCF is set to ALWAYS?

Reply:
We will have to make the change in our light fork then. Keeping this change does not affect the test even with UCF=always.